releases/release-1.31/release-notes/maps/pr-118148-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 118148
+releasenote:
+  text: The kube-scheduler exposes /livez and /readz for health checks that are in
```

```suggestion
  text: Exposed the kube-scheduler "/livez" and "/readz" endpoints for health checks that are
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653149908